applications/nrf_desktop/CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ set(mcuboot_CONF_FILE
   ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/mcuboot_${CMAKE_BUILD_TYPE}.conf
   )
 
+set(PM_STATIC_YML_FILE
```

Comment:
Could we add a comment here describing why we want the partitions declared statically instead of dynamically?

Reply:
thanks, will add it

Reply:
Why do you want consistency and what specifically do you want to be consistent ?

Reply:
The comment about DFU should be in code, not in the git commit message.